File: net/src/Sdl.Web.PublicContentApi/ContentModel/Pagination.cs
namespace Sdl.Web.PublicContentApi.ContentModel
{
    /// <summary>
    /// Cursor-Based Pagination
    /// </summary>
    public class Pagination : IPagination
    {
        /// <summary>
        /// Return the first n number of results
        /// </summary>
        public int First { get; set; }

        /// <summary>
        /// Start returning results after the specified cursor. 
        /// <remarks>Place your cursor on edges in your query.</remarks>
        /// </summary>
        public string After { get; set; }
    }
}

File: java/src/main/java/com/sdl/web/pca/client/ContentModel/Pagination.java
package com.sdl.web.pca.client.contentmodel;

public class Pagination implements IPagination {
    public int First;
    public String After;

    @Override
    public int getFirst() {
        return First;
    }

    @Override
    public void setFirst(int first) {
        First = first;
    }

    @Override
    public String getAfter() {
        return After;
    }

    @Override
    public void setAfter(String after) {
        After = after;
    }
}
